components/Toolbar/FilterInput.js

```diff
@@ -3,8 +3,8 @@ import PropTypes from "prop-types";
 import { Filter } from "patternfly-react";
 
 class FilterInput extends React.Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
```

Comment:
How is this change related?

Reply:
I had been considering setting state from props in the constructor and needed this then but you're correct I don't any more. But, I was reading about why props need to be passed to the constructor and super and came across [this section of the docs](https://reactjs.org/docs/state-and-lifecycle.html#adding-local-state-to-a-class) which says to always bind props to the constructor and super. It doesn't provide a reason but I think it is so this.props is initialized by the constructor and not after it. What are your thoughts?

Reply:
Yes, basically if you don't call super(props) in the contructor, this.props would still be undefined between the super call and the end of your constructor. But then react does this this.props = props anyway, so you can access the props from render and other methods later on.
So coming back to this specific change, I don't think it's anyhow adding some functionality. :)

Reply:
Okay I have removed it